indico/modules/rb/controllers/backend/misc.py

```diff
@@ -53,6 +53,16 @@ def _process(self):
                        privacy_policy_html=privacy_policy_html)
 
 
+class RHNotificationSettings(RHRoomBookingBase):
+    def _process(self):
+        return jsonify(notification_before_days=str(rb_settings.get('notification_before_days')),
+                       notification_before_days_weekly=str(rb_settings.get('notification_before_days_weekly')),
+                       notification_before_days_monthly=str(rb_settings.get('notification_before_days_monthly')),
+                       end_notification_daily=str(rb_settings.get('end_notification_daily')),
+                       end_notification_weekly=str(rb_settings.get('end_notification_weekly')),
+                       end_notification_monthly=str(rb_settings.get('end_notification_monthly')))
```

Comment:
We can use the schema here (like I mentioned on Mattermost), and I would not return this data as strings... If the placeholders need to be strings, you can just wrap them in `String(...)` in the JS

Reply:
Fixed ;)